File: 151004/Ivanenko/DC_Lab1/DC_Lab1/Models/Tweet.cs
using System;
using System.Collections.Generic;
using DC_Lab1.Models;

namespace DC_Lab1;

public partial class Tweet
{
    public int Id { get; set; }

    public int? EditorId { get; set; }

    public string? Title { get; set; }

    public string? Content { get; set; }

    public string? Created { get; set; }

    public string? Modified { get; set; }

    public virtual Editor? Editor { get; set; }

    public virtual ICollection<Post> Posts { get; set; } = new List<Post>();
}

File: 151002/Bybikov/FirstRestSolution/src/main/java/com/example/rv/impl/tweet/Tweet.java
package com.example.rv.impl.tweet;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Tweet {
    Long id;
    Long editorId;
    String title;
    String content;
    String created;
    String modified;
}
